package.json

```
@@ -12,6 +12,7 @@
     "clean": "rimraf dist",
     "watch": "cross-env NODE_ENV=development rollup -c --watch",
     "copy-test-files": "copyfiles test/**/* dist -f",
+    "test-jest": "jest",
     "test-serve": "serve .",
     "test-embed-serve": "serve . -l 3333",
     "test": "concurrently pnpm:watch pnpm:copy-test-files pnpm:test-serve pnpm:test-embed-serve",
```

The `test` command should run the automated tests now that we have them.

```suggestion
    "test": "jest",
```